coord/src/module.c

```diff
@@ -702,7 +707,7 @@ searchResult *newResult(searchResult *cached, MRReply *arr, int j, searchReplyOf
   int fieldsOffset = offsets->firstField;
   int payloadOffset = offsets->payload;
   int sortKeyOffset = offsets->sortKey;
-  searchResult *res = cached ? cached : malloc(sizeof(searchResult));
+  searchResult *res = cached ? cached : rm_malloc(sizeof(searchResult));
```

Comment:
please make the change of moving from the generic allocation function to `rm_alloc` in a separate PR.